include/svnlook.php

```diff
@@ -861,6 +861,50 @@ function listFileContents($path, $rev = 0, $peg = '') {
 		}
 	}
 
+	// }}}
+	
+	// {{{ listReadmeContents
+	//
+	// Parse the readme.md file
```

Comment:
Attention, it will not parse `readme.md` on POSIX, but `README.md`.

Reply:
Oh yes... Corrected now.